apiserver/facades/agent/secretsmanager/secrets.go

```diff
@@ -412,6 +420,51 @@ func (s *SecretsManagerAPI) WatchSecretsChanges(args params.Entities) (params.St
 	return results, nil
 }
 
+// WatchObsoleteRevisions sets up a watcher to notify of changes to obsolete secret revisions for the specified secrets.
+func (s *SecretsManagerAPI) WatchObsoleteRevisions(args params.Entities) (params.StringsWatchResults, error) {
+	results := params.StringsWatchResults{
+		Results: make([]params.StringsWatchResult, len(args.Entities)),
+	}
+	one := func(arg params.Entity) (string, []string, error) {
```

Comment:
Can we name this better? I would be keen to just make it it's own function.

Reply:
this follows established patterns used approx 40 times elsewhere in out apiserver code :-)
would prefer to be consistent

Reply:
ok (sad panda)